### PR TITLE
[17.0][MIG] product_net_weight : Migration to 17.0

### PR DESCRIPTION
**Pull request by remihb**
Duplicate of #1520 

The PR #1520 has issues with remaining `attrs` in view causing a Raise at module installation
```
odoo.tools.convert.ParseError: while parsing /home/odoo/odoo/parts/product-attribute/product_net_weight/views/view_product_template.xml:9
Depuis 17.0, les attributs "attrs" et "states" ne sont plus utilisés.
Vue : product.template form dans product_net_weight/views/view_product_template.xml
```

### CODE REVIEW

**Review by ioans73 — APPROVED** [review]
Functional review

**Comment by rousseldenis** [comment]
@peeranutjobgo Do you mind if we switch to this ?

/ocabot migration product_net_weight

**Review by rousseldenis — APPROVED** [review]
Code review

File: product_net_weight/views/view_product_template.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                    for="net_weight"
+                    invisible="product_variant_count > 1 and not is_product_variant"
+                />
+                <div
+                    class="o_row"
+                    name="net_weight"
+                    invisible="product_variant_count > 1 and not is_product_variant"
+                >
+                    <field name="net_weight" class="oe_inline" />
+                    <span><field name="weight_uom_name" /></span>
```
Comment (drewes) [inline product_net_weight/views/view_product_template.xml:27]: Would recommend to remove the <span> elements to align units with the standard product form

File: product_net_weight/views/view_product_product.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    <record id="view_product_product_form" model="ir.ui.view">
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_variant_easy_edit_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//label[@for='weight']" position="before">
+                <label for="net_weight" />
+                <div class="o_row">
+                    <field name="net_weight" />
+                    <span><field name="weight_uom_name" /></span>
```
Comment (drewes) [inline product_net_weight/views/view_product_product.xml:17]: Would remove <span> to align unit with standard product form (and in additions below)

**Comment by MarcoCalcagni** [comment]
@rousseldenis @pedrobaeza 
codecov/patch In progress pls note that is Started 1m 24d 2h 5m 37s ago. 

pls merge

**Comment by pedrobaeza** [comment]
What about the last 2 pending comments?

**Review by alexis-via — APPROVED** [review]

File: product_net_weight/views/view_product_template.xml (hunk 1 above)
Comment (rf-ife) [reply product_net_weight/views/view_product_template.xml:27]: @drewes a new PR where the issue is fixed https://github.com/OCA/product-attribute/pull/1735/files

**Review by bosd — APPROVED** [review]

**Comment by rvalyi** [comment]
@remihb I extracted the extra layout form fix from #1735
and sent a PR to you here https://github.com/osiell/product-attribute/pull/1

If you merge it quick, I guess we can merge your PR here.
If not, then I made a new PR in the OCA with the adicional fix: https://github.com/OCA/product-attribute/pull/1740

**Review by Borruso — APPROVED** [review]

**Comment by rvalyi** [comment]
supersedeed by #1740

**Pull request closed without merging**